### PR TITLE
Fix PackageUtils.get_binary_url when trying to install a non-uploaded package

### PR DESCRIPTION
**Pull request by Zopolis4**
oops

### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/Zopolis4/chromebrew.git CREW_BRANCH=ansc crew update
```

### CODE REVIEW

File: lib/const.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Defines common constants used in different parts of crew
 require 'etc'
 
-CREW_VERSION = '1.45.0'
+CREW_VERSION = '1.45.1'
```
Comment (uberhacker) [inline lib/const.rb:5]: This should be 1.49.9 since the last version was 1.49.8.

**Comment by uberhacker** [comment]
Let's revert the last commit and resolve conflicts correctly the next time.  See #10253.

File: lib/const.rb (hunk 1 above)
Comment (Zopolis4) [reply lib/const.rb:5]: i don't understand what you mean-- the current version is 1.50.0 in this PR, and the previous PR went from 1.49.9 to 1.50.0

Comment (uberhacker) [reply lib/const.rb:5]: This PR downgraded crew from 1.49.9 to 1.45.0.

Comment (Zopolis4) [reply lib/const.rb:5]: ohhhh i can't read and i did the version number wrong oops

**Pull request closed without merging**